scripts/netlify.sh

```diff
@@ -5,10 +5,11 @@ set -e # exit when error
 rm -rf ./netlify-dist
 mkdir -p ./netlify-dist/examples
 
-# build community website
-(cd community-website && yarn)
-(cd community-website && ROOT_PATH=$ROOT_PATH yarn docs:build)
-mv ./community-website/docs/* ./netlify-dist/
+# TODO: remove community website
+# # build community website
+# (cd community-website && yarn)
+# (cd community-website && ROOT_PATH=$ROOT_PATH yarn docs:build)
+# mv ./community-website/docs/* ./netlify-dist/
 
```

Comment:
now this will actually remove the community website (without redirects) once we merge the PR from v4 to master

Reply:
Who should we get in touch to make this redirection happen?

Reply:
We can undo this in another PR as you only removed the build command, but I think as long as we never update gh-pages, we are fine?